<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Input #2:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEqual">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="icon">
     <iconset theme="=">
      <normaloff>../../OneDrive/Pictures</normaloff>../../OneDrive/Pictures</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="result">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSrt">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>âˆš</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDiv">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>/</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Input #1:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMinus">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPlus">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="icon">
     <iconset theme="+">
      <normaloff>../../OneDrive/Pictures</normaloff>../../OneDrive/Pictures</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input1">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button0">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Operation:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input2">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClear">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMulti">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDot">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: px;
}

QPushButton:pressed {
	background-color: #808080;
	border: 1px solid gray;
	padding: px;
}

</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="op">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
